src/include/platform/CHIPDeviceConfig.h

```diff
@@ -1237,7 +1237,7 @@
  */
 #define CHIP_DEVICE_CONFIG_DISCOVERY_DISABLED 0
 #define CHIP_DEVICE_CONFIG_DISCOVERY_NO_TIMEOUT -1
-#define CHIP_DEVICE_CONFIG_EXTENDED_DISCOVERY_TIMEOUT_SECS (15 * 60)
+#define CHIP_DEVICE_CONFIG_EXTENDED_DISCOVERY_TIMEOUT_SECS (24 * 60 * 60)
```

Comment:
Why is this change needed? I think the spec recommends limiting announcements by default to up to 15 min.

Reply:
Can you share where you see that? My understanding is that the 15 minute recommendation is for commissioning mode and not extended discovery.

Reply:
> The default behavior for Commissionable Node Discovery SHOULD default to limiting announcement as
defined in Section 5.4.2.3, “Announcement Duration”
It doesn't specifically say anything about the commissioning mode and the text is part of the "Extended discovery" section so my understanding is that the recommendation applies to the extended discovery as well.

Reply:
ok, I'm not sure on that sentence either but happy to keep the default at 15 mins and so I changed this back and added an override for the tv-app.